core/src/main/java/org/apache/iceberg/rest/RESTCatalogProperties.java

```diff
@@ -22,9 +22,17 @@ public final class RESTCatalogProperties {
 
   private RESTCatalogProperties() {}
 
+  /** The name of the REST catalog startup parameter to configure default snapshot loading mode. */
   public static final String SNAPSHOT_LOADING_MODE = "snapshot-loading-mode";
+
   public static final String SNAPSHOT_LOADING_MODE_DEFAULT = SnapshotMode.ALL.name();
 
+  /**
+   * The name of the query parameter to be used in the load table endpoint of the REST catalog
+   * server to set the snapshot loading mode.
+   */
+  public static final String SNAPSHOT_MODE_PARAM_NAME = "snapshots";
```

Comment:
so this one is actually a query param and not a catalog property, therefore I don't think it should be part of this class